frontend/src/routes/moods/my-mood/createMood/+page.svelte

```diff
@@ -89,7 +95,9 @@
                 <label
                   for="2"
                   class="justify-center items-center flex flex-col
-                  {selectedMoodType === "2" ? "text-sm font-medium selected-label": "text-sm font-medium"}"
+                  {selectedMoodType === "2"
```

Comment:
I hate these if statements. But no judgement :P

Reply:
okay and? :D <3